osu.Framework/Testing/Drawables/Steps/StepSlider.cs

```diff
@@ -64,19 +66,21 @@ public StepSlider(string description, T min, T max, T start)
             currentNumber.SetDefault();
         }
 
-        protected override void OnDragEnd(DragEndEvent e)
+        protected override bool OnMouseDown(MouseDownEvent e)
         {
-            var flash = new Box
+            if (e.Button.HasFlagFast(MouseButton.Right))
```

Comment:
`MouseButton` isn't a flag enum, so using `HasFlagFast` in this context is actively misleading.

```suggestion
            if (e.Button == MouseButton.Right)
```

This should also probably not fire when a drag is in progress, because it doesn't really work that well in that case.